### PR TITLE
Angular: use single version in dependencies

### LINKED ISSUE #3836: Angular: use single version in dependencies
Following https://github.com/jhipster/jhipster-lite/pull/3835 and several other PRs, it would be cool to:
- add a single version of angular in dependency package.json
- use this same version for all other libs

Pro:
- less dependabot upgrade
- no need to manually do a PR to upgrade Angular

### PR DESCRIPTION
**Pull request by murdos**
Fixes #3836

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java [hunk 1]
```diff
@@ -1,10 +1,103 @@
...
+  public VersionSource versionSource() {
+    return versionSource;
+  }
+
+  public Optional<PackageName> versionPackageName() {
+    return versionPackageName;
+  }
+
+  @Override
+  public boolean equals(Object obj) {
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java:34]: You can exclude it from coverage

File: src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java [hunk 2]
```diff
@@ -1,10 +1,103 @@
...
+      return true;
+    }
+    if (obj == null || obj.getClass() != this.getClass()) {
+      return false;
+    }
+    var that = (PackageJsonDependency) obj;
+    return Objects.equals(this.packageName, that.packageName) && Objects.equals(this.versionSource, that.versionSource);
+  }
+
+  @Override
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java:45]: Same, exclude it

File: src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java [hunk 3]
```diff
@@ -1,10 +1,103 @@
...
+  public int hashCode() {
+    return Objects.hash(packageName, versionSource);
+  }
+
+  @Override
```
Comment (pascalgrimaud) [inline src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java:50]: Same, exclude it too

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

File: src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java (hunk 3 above)
Comment (murdos) [reply src/main/java/tech/jhipster/lite/module/domain/packagejson/PackageJsonDependency.java:53]: Done!

**Review by pascalgrimaud — APPROVED** [review]

**Pull request merged**